3기/LeeGaYeon/week2_mission/css/style.css

```
@@ -0,0 +1,319 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: 'Inter';
+  src: url(https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css) format(('woff'));
+}
+
+@font-face {
+  font-family: 'Inter';
+  src: url('https://fonts.googleapis.com/css2?family=Inter&display=swap') format('woff');
+  unicode-range: U+0041-005A, U+0061-007A;
+}
+
+h2 {
+  color: #2d3436;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter';
+  font-size: 16px;
+  line-height: 160%;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#wrap {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  width: 100%;
+  height: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 120px;
+  min-height: 100px;
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+}
+
+h1 {
+  font-size: 20px;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100px;
+  background-color: #2d3436;
+}
+
+.footer-content {
+  color: #dfe4ea;
+}
+
+main {
+  flex: 1;
+  display: flex;
+}
+
+article.about {
+  flex: 1;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 120px;
+}
```

어차피 class명이 있기 때문에 태그명과 함께 중첩선택을 하지 않는게 좋아요.
이렇게 했을 때 이름을 붙여뒀는데 태그가 수정되면 결국 CSS를 또 수정해야 하는 불편함이 있어요!

```suggestion
.about {
  flex: 1;
  position: relative;
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: 120px;
}
```